commands/container-registry/registry/post.go

```diff
@@ -0,0 +1,111 @@
+package registry
+
+import (
+	"context"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/constants"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/core"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/printer"
+	sdkgo "github.com/ionos-cloud/sdk-go-container-registry"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var regPostProperties = sdkgo.PostRegistryProperties{}
+
+func RegPostCmd() *core.Command {
+	cmd := core.NewCommand(
+		context.TODO(), nil, core.CommandBuilder{
+			Namespace:  "container-registry",
+			Resource:   "registry",
+			Verb:       "create",
+			Aliases:    []string{"c"},
+			ShortDesc:  "Create a registry",
+			LongDesc:   "Create a registry to hold container images or OCI compliant artifacts",
+			Example:    "ionosctl container-registry registry create",
+			PreCmdRun:  PreCmdPost,
+			CmdRun:     CmdPost,
+			InitClient: true,
+		},
+	)
+
+	cmd.Command.Flags().StringSlice(constants.ArgCols, nil, printer.ColsMessage(allCols))
+	_ = cmd.Command.RegisterFlagCompletionFunc(
+		constants.ArgCols,
+		func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return allCols, cobra.ShellCompDirectiveNoFileComp
+		},
+	)
+	cmd.AddStringFlag(FlagName, "n", "", "Specify the name of the registry", core.RequiredFlagOption())
+	cmd.AddStringFlag(FlagLocation, "", "", "Specify the location of the registry", core.RequiredFlagOption())
+
+	cmd.AddStringSliceFlag(
+		FlagRegGCDays, "", []string{}, "Specify the garbage collection schedule days",
+	)
+	_ = cmd.Command.RegisterFlagCompletionFunc(
+		FlagRegGCDays,
+		func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return []string{
+				"Modnday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday",
+			}, cobra.ShellCompDirectiveNoFileComp
+		},
+	)
+	cmd.AddStringFlag(FlagRegGCTime, "", "", "Specify the garbage collection schedule time of day")
```

Comment:
would be nice to tell the users what time format we expect. afaik it is 00:00:00? could even add a few completer options in that format, but that's up to you